src/components/providers/authentication-provider.tsx

```diff
@@ -80,8 +81,8 @@ const AuthenticationProvider = ({ children }: Props) => {
    * Sets the logged in timebank person from keycloak ID into global state
    */
   const getPersonsList = async () => {
-    const fetchedPersons = await personsApi.listPersons({ active: true });
-    setPersons(fetchedPersons);
+    const fetchedPersons = await usersApi.listUsers({ active: true });
```

Comment:
I dont think "active: true" is needed. (?)

Reply:
I agree. Made changes, also named const to fetchedUsers

Also fixed other issue in
src/components/providers/authentication-provider.tsx
line 291 added draft.false to remove problem there